playgrounds/react/src/examples/form/components/form.tsx

```diff
@@ -65,10 +45,12 @@ export function Form({ children, onSubmit }: FormProps) {
 
   return (
     <FormContext.Provider value={formContextValue}>
-      <form onSubmit={handleSubmit}>
-        {children}
-        <input type="submit" value="Submit" />
-      </form>
+      <ValidationContext.Provider value={validationContextValue}>
```

Comment:
Suggestion to the next version of validation.
Move ValidationContext.Provider into new Validator component and use it here.